src/main/java/bio/terra/cli/businessobject/Workspace.java

```diff
@@ -164,6 +159,19 @@ public static Workspace get(UUID id) {
     return workspace;
   }
 
+  /** Fetch an existing workspace by userFacingId, with resources populated */
+  public static Workspace get(String userFacingId) {
+    // call WSM to fetch the existing workspace object and backing Google context
+    WorkspaceDescription loadedWorkspace =
+        WorkspaceManagerService.fromContext().getWorkspaceByUserFacingId(userFacingId);
+    logger.info("Loaded workspace: {}", loadedWorkspace);
+
+    // convert the WSM object to a CLI object
```

Comment:
this bit is shared with the overload for `get()`. You could pull out a `getPopulatedWorksapce(WorkspaceDescription desc)` and use it in both places.

Reply:
Done